src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -207,10 +213,12 @@ class SourcesTree extends Component<Props, State> {
     if (!nodeHasChildren(item)) {
       const obj = item.contents.get("id");
       const source = sources.get(obj);
+      const original = isOriginalId(source.id);
       return (
         <img
           className={classnames(
             getSourceClassnames(source.toJS()),
+            { original },
```

Comment:
I like the idea of doing something different with original files but since we don't have any style to go with it, we can remove the className for now.